vue.config.js

```diff
@@ -1,13 +1,5 @@
-const apiMocker = require('connect-api-mocker');
-
 module.exports = {
   lintOnSave: true,
-  devServer: {
```

Comment:
Do we need to remove the mock data in favour of using the docker container? I thought it was a nice option to mock the api like this. Maybe we should turn it into a setting in the `.env` files.

Reply:
We can, but then we'd have to make the mocks match the actual API.

Probably best to discuss various implementations on thursday.